### PR TITLE
OKTA-355288 : Update Authenticators Admin API docs to include the info and samples for the allowedFor field

### PR DESCRIPTION
**Pull request by rameshbhojan-okta**
## Description:
- **Update Authenticators Admin API docs to include the info and samples for the allowedFor field**
- **Is this PR related to a Monolith release? - 2021.01.0** 

### Resolves:

* [OKTA-355288](https://oktainc.atlassian.net/browse/OKTA-355288)

### CODE REVIEW

**Review by haoyuli-okta — APPROVED** [review]
LGTM double-checked the API response.

File: packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md [hunk 1]
```diff
@@ -528,6 +534,7 @@ The Authenticator object defines the following properties:
 | `lastUpdated` | String (ISO-8601)                                               | Timestamp when the Authenticator was last modified.                             |
 | `name`        | String                                                          | Display name of this Authenticator.                                             |
 | `type`        | String (Enum)                                                   | The type of Authenticator. Possible values: `password`, `security_question`, `phone`, `email`, `security_key`                            |
+| `allowedFor`        | String (Enum)                                                   | The allowed types of usages for the Authenticator. Possible values: `recovery`, `sso`, `any`, `none`                            |
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md:537]: One suggestion: as per my Slack question, if allowed types are limited, include the serial "and" and remove "possible"; that is, "Values include `recovery`, `sso`, `any`, and `none`"
(Same comment for the `type` property above `allowedFor`)

**Review by brianduffield-okta — APPROVED** [review]
All looks good. One suggestion, which I believe Ramesh is updating.

File: packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md (hunk 1 above)
Comment (rameshbhojan-okta) [reply packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md]: Done!

**Comment by brianduffield-okta** [comment]
Merging PR into 2021.01.0 Release Branch

**Pull request merged**